### PR TITLE
#40 added health and safety guidelines section

### PR DESCRIPTION
**Pull request by NavAneEth777**
Added health and safety guidelines in header section..

### CODE REVIEW

**Comment by anistark** [comment]
The [page link from menu](https://inpycon2024-git-fork-navaneeth777-5bfdb9-python-indias-projects.vercel.app/2024/health-and-safety/) isn't opening for me

**Comment by bhansa** [comment]
@anistark you can access this for now, ignore the broken assets: https://inpycon2024-git-fork-navaneeth777-5bfdb9-python-indias-projects.vercel.app/health-and-safety/

This is due to vercel deployment, we need to change the settings to use pathprefix while build.
@palnabarun can you help here to update the build command for vercel to:
`npx @11ty/eleventy --pathprefix=/2024/`

File: src/_pages-content/health-and-safety.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+This does represent a change from the most recent two PyCon India events; a verified vaccination status was required to attend both PyCon India 2022 and 2023. We have made the decision not to require vaccination for a few reasons, including the increasingly wide variation in availability and recommendations for vaccinations and boosters globally, changes over time in the relevance of vaccination status in risk calculation, as well as the significant cost of the verification procedure which we can redirect to other safety measures.
+
+Like the rest of this policy, we may review this stance and consider adding additional precautions should the situation change (see final paragraph of this policy).
+
+### RAPID COVID-19 TESTS ARE ENCOURAGED
+We encourage PyCon India attendees to use at-home/antigen/rapid tests before they arrive and while they are at PyCon India. We will be providing free additional rapid tests as supplies permit to anyone who needs them in the staff office.
+
+### ATTEND ONLINE
```
Comment (anistark) [inline src/_pages-content/health-and-safety.md]: We don't have online.

File: src/_pages-content/health-and-safety.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+We are going to continue investing in making PyCon India Online (#PyConOX) an excellent option for folks who don’t feel comfortable attending a masked event, or find travel to Pittsburgh inconvenient, or want to hang with the awesome PyConOX volunteer team, or who would prefer online for any reason at all.
+
+### EASY REFUNDS
+We expect people to stay home if they may be contagious. If you are feeling sick or exhibiting symptoms of COVID-19, or test positive for COVID-19, prior to the start of the conference, or on any day of the conference, please contact us at pycon-reg@python.org, and we will exchange your ticket type for an online ticket or grant you a refund. Onsite attendees will be able to watch talks online while they recover, wait for the results of a Covid test or sort out any kind of unexpected allergy or “weird cold.”
```
Comment (anistark) [inline src/_pages-content/health-and-safety.md]: There's separate window for cancellations. Please refer to that.

File: src/_pages-content/health-and-safety.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+layout: content.liquid
+permalink: /health-and-safety/
+---
+### WHY WE (STILL) HAVE A HEALTH & SAFETY POLICY?
+
+We are nice people and many of us and our fellow community members can’t attend without health and safety guidelines in place. We want PyCon India to be an event that everyone feels safe attending. PyCon India is exceptional in that we have people attending from all over the world—last year we had people attending from 75 different countries! We love that people bring their unique perspectives… but they also bring their local germs and Covid strains. We’re masking to keep our immuno-compromised friends safe at PyCon India and to ensure that those of us who share their homes with immuno-compromised people can also consider joining us without quarantining when they return home.
+
+We are going to be doing some analysis on the airflow and refresh rates while we are in the convention center this year to see if there are some spaces that are well-ventilated enough to be indistinguishable from an outdoor situation for next PyCon India 2025. But for 2024, we are sticking with masking in all of the conference spaces.
+
+### MASKS ARE REQUIRED
```
Comment (anistark) [inline src/_pages-content/health-and-safety.md:12]: "Encouraged"

Please modify the rest content accordingly.

**Review by anistark — CHANGES_REQUESTED** [review]

File: src/_pages-content/health-and-safety.md (hunk 1 above)
Comment (NavAneEth777) [reply src/_pages-content/health-and-safety.md]: i have to remove that?

File: src/_pages-content/health-and-safety.md (hunk 3 above)
Comment (NavAneEth777) [reply src/_pages-content/health-and-safety.md:12]: please give me the info to add..

**Comment by NavAneEth777** [comment]
once check

File: src/_pages-content/health-and-safety.md (hunk 1 above)
Comment (anistark) [reply src/_pages-content/health-and-safety.md]: Yes of course. We don't have online. Did we mention about online?

File: src/_pages-content/health-and-safety.md (hunk 3 above)
Comment (anistark) [reply src/_pages-content/health-and-safety.md:12]: We don't enforce if someone is wearing masks or not. It's advised to wear.

**Comment by sayanchowdhury** [comment]
I think it would be better to collab on a rise up pad / or a hackmd doc. Quicker response

**Comment by bhansa** [comment]
@NavAneEth777 can you work on it this week, please share the risepad etc. for faster reviews.

**Comment by anistark** [comment]
This is done. Closing the PR.

**Pull request closed without merging**